common/VotingUtils.js

```diff
@@ -101,7 +102,7 @@ const constructReveal = async (request, roundId, web3, signingAccount, votingCon
     identifier: request.identifier,
     time: request.time,
     price: vote.price.toString(),
-    salt: web3.utils.hexToNumberString("0x" + vote.salt.toString())
```

Comment:
I didn't test this, but I think this PR could also have been resolved by removing this `.toString()` in the reveal process here...